test/validators/htmlValidator.js

```diff
@@ -0,0 +1,44 @@
+require('chai').should();
+const expect = require('chai').expect;
+
+const HTMLValidator = require('../../lib/validators/html-validator');
+const RouteInfo = require('../../lib/models/route-info');
+const parseResources = require('../../lib/parsers/resource-parser');
+const parseHTML = require('../../lib/parsers/html-parser');
+
+const YAML = require('yaml');
+const fs = require('fs');
+const Path = require('path');
+
+const CONFIG_FILE = '../resources/config-missing-html.yaml';
+
+describe('HTMLValidator', function () {
+  {
+    before(function () {
+      const fileAbsPath = Path.resolve(__dirname, CONFIG_FILE);
+      const file = fs.readFileSync(fileAbsPath, 'utf8');
+      this.yamlData = YAML.parse(file);
+      this.resources = parseResources(this.yamlData['x-walder-resources'], Path.dirname(fileAbsPath));
+    });
+
+    describe('# Files', function () {
+      it('Should return \'undefined\' when all HTML and template files are available', function () {
```

Comment:
If the `'` is replaced with backticks then `'` does not have to be escaped around undefined.

Reply:
Copied code again. Fixed, also at copy source. FWIW...